internal/rego/version/rego_default_v1.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2024 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+//go:build !opa_rego_v0
```

Comment:
[nit] the other file has the old-style `// +build` comment, too. I think we only need `//go:build` these days, don't we? 🤔